cpp/include/cudf/transform.hpp

```diff
@@ -111,5 +111,30 @@ std::pair<std::unique_ptr<cudf::column>, std::unique_ptr<cudf::column>> encode(
   cudf::column_view const& input,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Creates a boolean column from given bitmask.
+ *
+ * Each bit represents one value, 0 - false and 1 - true.
+ * It is user's responsibility to make sure the bitmask agrees
+ * with given offset and length.
```

Comment:
```suggestion
 * Returns a `bool` for each bit in `[begin_bit, end_bit)`. If bit `i` in least-significant bit numbering is set (1), then element `i` in the output is `true`, otherwise `false`. 
```